### PR TITLE
Cronjob eco issues

### PR DESCRIPTION
**Pull request by YustinaKvr**
1) Del cronjobs for deleted scripts
2) Add cronjob for ecosystem_issues.py script
3) Fix kustomization.yaml

### CODE REVIEW

**Review by vladimirvshivkov — DISMISSED** [review]

File: kubernetes/grafana-docs-monitoring/base/cronjob-ei.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: grafana-docs-monitoring-failed-zuul
```
Comment (vladimirvshivkov) [inline kubernetes/grafana-docs-monitoring/base/cronjob-ei.yaml:4]: change failed zuul to eco-issues here

File: kubernetes/grafana-docs-monitoring/base/cronjob-ei.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+                - mountPath: "/etc/vault"
+                  name: "vault-agent-config"
+                - mountPath: "/secrets"
+                  name: "secrets"
+                - mountPath: "/var/run/secrets/tokens"
+                  name: "k8-tokens"
+                  readOnly: true
+
+          containers:
+            - name: grafana-docs-monitoring-failed-zuul
```
Comment (vladimirvshivkov) [inline kubernetes/grafana-docs-monitoring/base/cronjob-ei.yaml:41]: and here

**Review by vladimirvshivkov — CHANGES_REQUESTED** [review]

File: kubernetes/grafana-docs-monitoring/base/cronjob-ei.yaml (hunk 2 above)
Comment (YustinaKvr) [reply kubernetes/grafana-docs-monitoring/base/cronjob-ei.yaml]: fixed

File: kubernetes/grafana-docs-monitoring/base/cronjob-ei.yaml (hunk 1 above)
Comment (YustinaKvr) [reply kubernetes/grafana-docs-monitoring/base/cronjob-ei.yaml]: fixed

**Review by vladimirvshivkov — APPROVED** [review]

**Pull request merged**